test-utils/src/performance/history-store/main.rs

```diff
@@ -0,0 +1,177 @@
+    }
+    println!("Done generating txns. ");
+
+    println!("Adding txns to history store");
+    let start = Instant::now();
+
+    let batches_per_round = batches / rounds;
+    let blocks_per_round = number_of_blocks / rounds;
+    println!(
+        "Number of rounds: {}, batches per round {} blocks per round {}",
+        rounds, batches_per_round, blocks_per_round
+    );
+
+    for round in 0..rounds {
+        let round_start = Instant::now();
+        let mut txn = env.write_transaction();
+
+        for block_number in 0..blocks_per_round {
+            history_store.add_to_history(
+                &mut txn,
+                (round * blocks_per_round) + (1 + block_number),
+                &txns_per_block[((round * blocks_per_round) + block_number) as usize],
+            );
+        }
+
+        let round_duration = round_start.elapsed();
+
+        println!(
+            "...{:.2}s to process round {}  ",
+            round_duration.as_millis() as f64 / 1000_f64,
+            1 + round
+        );
+        txn.commit();
+    }
+
+    let duration = start.elapsed();
+
+    let db_file_size = fs::metadata(db_file.to_str().unwrap()).unwrap().len();
+
+    println!(
+        "{:.2}s to add {} batches, {} tpb, DB size: {:.2}Mb, rounds: {}, total_txns: {}",
+        duration.as_millis() as f64 / 1000_f64,
+        batches,
+        num_txns,
+        db_file_size as f64 / 1000000_f64,
+        rounds,
+        num_txns * number_of_blocks
+    );
+
+    let mut txn = env.write_transaction();
+    println!("Pruning the history store..");
+    let start = Instant::now();
+
+    history_store.remove_history(&mut txn, Policy::epoch_at(1));
+
+    let duration = start.elapsed();
+
+    println!(
+        "It took: {:.2}s, to prune the history store",
+        duration.as_millis() as f64 / 1000_f64,
+    );
+}
+
+fn main() {
+    let args = Args::parse();
+
+    let policy_config = Policy::default();
+
+    let _ = Policy::get_or_init(policy_config);
+
+    let temp_dir = tempdir().expect("Could not create temporal directory");
+    let tmp_dir = temp_dir.path().to_str().unwrap();
+    let db_file = temp_dir.path().join("mdbx.dat");
+    log::debug!("Creating a non volatile environment in {}", tmp_dir);
+    let env = MdbxDatabase::new(tmp_dir, 1024 * 1024 * 1024 * 1024, 21).unwrap();
+
+    let history_store = if args.light > 0 {
+        println!("Exercising the light history store");
+        Box::new(LightHistoryStore::new(
+            env.clone(),
+            NetworkId::UnitAlbatross,
+        )) as Box<dyn HistoryInterface + Sync + Send>
+    } else {
+        println!("Exercising the history store");
+        Box::new(HistoryStore::new(env.clone(), NetworkId::UnitAlbatross))
+            as Box<dyn HistoryInterface + Sync + Send>
+    };
+
+    let rounds = args.rounds.unwrap_or(1);
+
+    history_store_performance(history_store, env, args.tpb, args.batches, rounds, db_file);
+
+    exit(0);
```

Comment:
You can delete the temp database file after the benchmark finishes:

```suggestion
    history_store_performance(history_store, env, args.tpb, args.batches, rounds, db_file);
    
    let _ = fs::remove_dir_all(temp_dir);

    exit(0);
```